src/applications/disability-benefits/all-claims/config/form0781/index.js

```diff
@@ -26,65 +26,65 @@ import {
  */
 export const form0781PagesConfig = {
   workflowChoicePage: {
-    path: 'additional-forms/mental-health-statement',
+    path: 'mental-health-form-0781/workflow',
     depends: formData => showForm0781Pages(formData),
     uiSchema: workflowChoicePage.uiSchema,
     schema: workflowChoicePage.schema,
   },
   manualUploadPage: {
     path:
-      'disability/file-disability-claim-form-21-526ez/additional-forms/mental-health-statement/upload',
+      'disability/file-disability-claim-form-21-526ez/mental-health-form-0781/upload',
```

Comment:
Nathan may address this in a future PR, but this path can be shortened to 'mental-health-form-0781/upload'.